pkg/daemon/update.go

```diff
@@ -1775,6 +1775,7 @@ func getSupportedExtensions() map[string][]string {
 		"kerberos":             {"krb5-workstation", "libkadm5"},
 		"kernel-devel":         {"kernel-devel", "kernel-headers"},
 		"sandboxed-containers": {"kata-containers"},
+		"sysstat":		{"sysstat"},
```

Comment:
spacing here probably needs to match

```suggestion
		"sandboxed-containers": {"kata-containers"},
		"sysstat":		        {"sysstat"},
```

Reply:
I was actually going back & forth on this spacing as the outputs were different in vi and vscode editors. Guess I should have trusted vscode here.